other/swap_all_odd_and_even_bits.dart

```diff
@@ -0,0 +1,9 @@
+//we are given an integer,the tast is to get its binary representation,swap all odd and even bits and print the new number
```

Comment:
Could you add some tests?
```suggestion
import 'package:test/test.dart';

//we are given an integer,the tast is to get its binary representation,swap all odd and even bits and print the new number
```